### PR TITLE
Remove /api/hour/begin_learn tracking, instead rely on ELB and GA

### PR DESCRIPTION
**Pull request by Hamms**

### CODE REVIEW

**Comment by breville** [comment]
Good code but turns out we have a cross-domain issue in which a page on hourofcode.com will be calling a launch_url on code.org via ajax.  Presumably this would require a server config change, and I'm not sure how @wjordan feels about that.

**Comment by Hamms** [comment]
Isn't the /learn page on `code.org`? Where are we using this on `hourofcode.com`?

**Comment by Hamms** [comment]
@breville @jeremydstone @philbogle talked with Asher and Alice, just awaiting final confirmation from Alice that this route is acceptable.

**Comment by Bjvanminnen** [comment]
Changes look ok to me, though this isn't my area of expertise.

File: pegasus/routes/hoc_routes.rb [hunk 1]
```diff
@@ -49,7 +49,7 @@
 get '/api/hour/begin_learn/:code' do |code|
   only_for ['code.org', 'csedweek.org', partner_sites].flatten
   pass unless tutorial = DB[:tutorials].where(code: code).first
-  launch_tutorial(tutorial, track_learn: true)
+  redirect tutorial[:url], 302
```
Comment (breville) [inline pegasus/routes/hoc_routes.rb:52]: @Hamms I know this function is no longer called, but since this didn't work the way we wanted, should we revert this change on staging?
